containers/matomo/config.ini.php

```diff
@@ -92,6 +92,7 @@
 Plugins[] = "UserId"
 Plugins[] = "CustomPiwikJs"
 Plugins[] = "WAIMatomoTheme"
+Plugins[] = "UsersManagerOnlyAdmin"
```

Comment:
Questa configurazione dovrebbe essere gestita come per gli altri plugin (es. TrackerHostname o LoginFilterIp) e non scritta direttamente in `config.ini.php`.